### ISSUE TITLE
How to limit the cpu usage to less than 500us with max_cpu_usage_ms?

### ISSUE THREAD

**Issue by shtse8** [EOSIO/eos#8257]
https://eosio.github.io/eosio.cdt/latest/classeosio_1_1transaction__header

most transaction is less than 1ms. but the minimum value of max_cpu_usage_ms is ms.

**Comment by taokayan** [comment]
looks like there's no straight forward way to do this. a protocol feature maybe needed.

**Comment by shtse8** [comment]
Yes, I think this is really in need because there are many slow BPs. sometimes, we just want to execute transactions with low cpu usage but not in emergency.
